homework/style.css

```diff
@@ -1,3 +1,104 @@
 .alert-error {
```

Comment:
Your CSS styling still has room for improvement.

Reply:
Changes made!